### ISSUE TITLE
Documentation on current programming status for future devs

### ISSUE THREAD

**Issue by Ezmeyers** [iupui-soic/mHBS_tracker#41]
Please find a guide attached on development, what has been done with development, what went wrong/has been tried with the pulling of data and suggestions for moving forward. This guide omits pieces that @tchilders95 has done, which he will upload as another document and attach here.

**Comment by tchilders95** [comment]
[mHBS_LearnerTracker_dev_guide.v2.docx](https://github.com/iupui-soic/mHBS_tracker/files/1195834/mHBS_LearnerTracker_dev_guide.v2.docx)
This is the dev guide combining my information with @Ezmeyers. Information I added regards the download functionality along with the logical flow of pdf and video population.

**Comment by shbucher** [comment]
This is great, @Ezmeyers @tchilders95!

**Comment by sunbiz** [comment]
Very impressed with the detail in the documentation. Excellent work!!

**Comment by shbucher** [comment]
Agreed with Dr, Saptarshi, a really stellar job. Well done, @Ezmeyers @tchilders95

**Comment by shbucher** [comment]
@Ezmeyers , please ensure that the MURI AY 2017-2018 students have read the documentation provided for the MURI Summer project by yourself and @tchilders95 , and that they are also documenting their efforts in a similar manner.  You and Taylor did a great job, and others should emulate your efforts!
